signac_dashboard/dashboard.py

```diff
@@ -338,7 +338,7 @@ def _job_search(self, query):
             )
             raise error
 
-    @lru_cache(maxsize=65536)
+    @lru_cache(maxsize=65536)  # arbitrarily set to 2**16
```

Comment:
This comment isn't related to the core aim of this PR. Can we keep the diff minimal on this PR and pull out extraneous changes since this PR is a significant change?